### ISSUE TITLE
Apple App Store issue with Construct 3 project

### ISSUE THREAD

**Issue by mageekm** [Scirra/Construct-3-bugs#1028]
## Problem description

Apple App store  requires my app to support iOS 8+ and Construct3 only supports iOS 9+
Also load failing asking for 64bit support.

*Enter a concise description of your problem here*

I develop on a windows machine and then transfer my app to a mac to do the final work of publishing to the app store.  I basically just take the defaults when exporting in Construct3.  I use the export manager to create a folder that I then transfer to a Mac running Xcode 9.1.  Export Manager requires me to support iOS 9+ even though my app runs fine at lower versions.  

All is well getting xcode up and running with the project no issues whatsoever.  I make sure my app is supporting "standard" architectures in settings and it builds and tests just fine.  Lots of warnings but no showstoppers.  I test the app on the simulator and devices that I have connected with no issues.  

My signing certificates, provisioning, team, and profiles are all correct.  I am able to archive and enter the organizer to upload to the app store.  This is where the shit hits the fan.  Keep in mind this is an upgrade to an app I already have on the store.  It was created in 2015 and is on version 3.  I am trying to upload version 4.  I have tried both using xcode and the application loader and getting the same results.

The app store goes through the process of uploading the app and then gives me an error stating that the app must support version 8 SDK or greater and 64bit processors.  It informs me to be sure to set the architectures to "standard" to encompass both 32 and 64bit arms. (This is where they were set and I double checked, no luck)  I have tried various settings in Xcode hoping to find a solution.  Setting the iOS support back to 8+ built cleanly but a few more warnings were generated.  
Here is the body of an email from Apple after my last attempt.
-------------------------------------
Dear developer,
We have discovered one or more issues with your recent delivery for "CampusQ". To process your delivery, the following issues must be corrected: 
Missing 64-bit support - iOS apps submitted to the App Store must include 64-bit support and be built with the iOS 8 SDK or later. We recommend using the default "Standard Architectures" build setting for "Architectures" in Xcode, to build a single binary with both 32-bit and 64-bit support. 
Once these issues have been corrected, you can then redeliver the corrected binary. 
Regards,
The App Store team
-------------------------------------------------------------
I am curious how any of your customers are using this feature (9+ only) of your export manager and actually publishing an app on the store.  Apple seems pretty clear here.

I have an exact version of the program on Construct 2 and am aware of the fact I can spec version iOS 8+ on export.  I have started that process but now see that I must create the cordova environment on my Mac as it is looking for xcodebuild on my windows machine.  Not cool!  I do not want to pay PhoneGap fees so I am purposely avoiding them.  None of my options look good at this point.  
What is the purpose of having an export function to xcode that is not supported by Apple?  I am sure this is not what you intended.

If you would like screenshots of my xcode build setting I can take snap shots for you.  Basically I have just taken the defaults and to my thinking is a waste of time.

## Attach a .c3p

This is not practical to me.  It gets you nowhere since this provides none of my signing certificates and so on.  I have published this app on Android Play Store (QDuku CampusQ)  and it is working just fine.  The problem is not the app but the requirements of the Apple App Store.  Even if I give you my .c3p, I can't see how you will reproduce the error with the store.  

Honestly I am trying to move to C3 but your unwillingness to support a few more versions back is pigheaded.  I had to drop back to C2 and work through the Cordova CLI to get it to the store with my Android version else I would have cut my market in half.  I get that the later versions allow you to do "magical" things with C3, but I am sure there are a lot of journeymen like me that are just fine with a subset of what C3 can do to get our work done.  Apple seems to agree.

Thanks for the support.

*A minimal Construct 3 project (.c3p) is required to be attached. Please see the guidelines*

## Steps to reproduce

1.
2.
3.

## Observed result

*What do you see happen?*

## Expected result

*What did you expect to happen instead?*

## Affected browsers

* Chrome: yes/no
* Firefox: yes/no
* Edge: yes/no
* Safari: yes/no

## System details

*If you see a crash report dialog, please copy and paste it to where it says "PASTE HERE" below. Otherwise please go to **Menu** -> **About** -> **Platform information** and paste that information there instead.*

<details><summary>View details</summary>

PASTE HERE

</details>

**Comment by shortercode** [comment]
I'm investigating this now, but the error seems to be more related to the fact the uploaded binary doesn't include 64 bit, "[...] with the iOS 8 SDK or later [...]" this seems to suggest it's fine to build with the iOS 9 SDK, and I would be surprised if apple prevented you from only targeting newer versions of iOS.

Please take a look at this in the meantime [xcode-missing-64-bit-support-with-standard-architectures-enabled](https://stackoverflow.com/questions/32514787/xcode-missing-64-bit-support-with-standard-architectures-enabled)

**Comment by AshleyScirra** [comment]
FWIW Construct 3 only supports iOS 9+ because it supports WKWebView, which guarantees your app runs ~3x faster.

**Comment by mageekm** [comment]
Hey Guys, 
Thanks for the quick response.  I really appreciate how hard your team is working on this.  Even on a holiday!  (US only).  Yes I missed the flag for "Build active architectures only".  Mine was set to "Yes".
Changing to "No" does solve the problem.  A word of caution, when set to "No" the Build option will fail under Product tab.  It pukes with a couple of errors in the Apple Mach-O linker.  The fix was to set to "No"  and use the Archive option instead.  This processed cleanly and got me to the Organizer, which then got me to the App Store.  Along the way I did get a message that an arm64 and arm7 version was cleanly built.  I seemed to have found all of the build flags that changed from 9 to 8 as the App Store is now happy.  I am now ready to start checking out the app on testflight, but I have some turkey waiting and a footee ball game to watch.  
Whew!  
I wonder what surprises lay ahead for me next, but as my daddy used to say "A faint heart never won a fair lady".
Thanks for the support.
Regards,
